charts/altinn-app/templates/ingressRoute.yaml

```diff
@@ -0,0 +1,48 @@
+{{- $environment := .Values.environment }}
+{{- $linkerd := .Values.linkerd }}
+
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRoute
+metadata:
+  name: {{ .Values.ingressRoute.name }}
+spec:
+  entryPoints:
+  {{- range $entryPoint := .Values.ingressRoute.entryPoints }}
+  - {{ $entryPoint }}
+  {{- end }}
+  routes:
+  {{- range $route := .Values.ingressRoute.routes }}
+  {{- $routeMatch := index $route.match $environment }}
+  - match: {{ $routeMatch }}
+    kind: {{ $route.kind }}
+    {{- if $route.priority }}
+    priority: {{ $route.priority }}
+    {{- end }}
+    services:
+    {{- range $service := $route.services }}
+    - name: {{ $service.name }}
+      port: {{ $service.port }}
+    {{- end }}
+    {{- if or $route.middlewares $linkerd.enabled }}
+    middlewares:
+    {{- if $route.middlewares }}
+    {{- range $middleware := $route.middlewares}}
+      - name: {{ $middleware.name }}
+    {{- end }}
+    {{- end }}
+    {{- if $linkerd.enabled }}
+      - name: {{ $linkerd.header }}
+    {{- end }}
+    {{- end }}
+  {{- end }}
+  {{- if .Values.ingressRoute.tls }}
+  tls:
+    {{- if .Values.ingressRoute.tls.options }}
+    options:
+      name: {{ .Values.ingressRoute.tls.options.name }}
```

Comment:
This is not supported config in traefik. traefik use option set as default for controller - you can override with anntations.